src/BMW_E65.cpp

```diff
@@ -152,16 +152,34 @@ void BMW_E65::Task10Ms()
 {
     if(CANWake)
     {
+        uint8_t data[8]; 
+        int rpm = 0;
         if (Ready())
         {
-            uint32_t data[2];
-            int rpm = MAX(750, revCounter) * 4; // rpm value for E65
-
-            data[0] = 0xff595f;
-            data[1] = 0x99800000 | rpm;
-            can->Send(0x0AA, data); //Send on CAN2
+            data[1] = 0x50 | AA1;  //Counter for 0xAA Byte 0
```

Comment:
The convention seems to be that the numbering scheme follows the byte position in the CAN message. So even though this is Data Byte 0, it is Message Byte 1, so it would be more consistent to write "Counter for 0xAA (Byte 1)" or just "Counter for 0xAA".